tests/summarizer/test_text_media_matcher.py

```diff
@@ -37,3 +37,31 @@ def test_text_media_matcher_matches_contents():
         (sentence_2, media_related_to_sentence_2)]
     assert processed_contents_dict['unused_contents'] == [
         media_related_to_sentence_1]
+
```

Comment:
Also add a case for when both media and text is empty?

Reply:
Ah yes. While this case is very unlikely to happen it is better to add a handler for that.